lighthouse-viewer/test/drag-and-drop-test.js

```diff
@@ -17,8 +15,12 @@ testHelpers.setupJsDomGlobals();
 
 const DragAndDrop = require('../app/src/drag-and-drop.js');
 
-function assertUIReset() {
-  assert.ok(!document.querySelector('.drop_zone').classList.contains('dropping'));
+function createCustomEvent(type, key, value) {
```

Comment:
nit: WDYT about the caller passing `{key: value}` to make it a little clearer below

Reply:
do you mean add a comment here?

Reply:
let's ignore this comment in favor of https://github.com/GoogleChrome/lighthouse/pull/9314#discussion_r299229402